packages/blade/docs/utils/coverge.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Utils/coverage" />
+
+## `getBladeCoverage`
+
+This utility function calculates the blade usage coverage in percentage of the DOM elements on a web page.
```

Comment:
We can expand this documentation when resolving https://github.com/razorpay/blade/issues/1302

Reply:
can we just add a link to chrome extension till then? because if someone new comes to documentation site and reads about coverage, they might think the only way to check coverage is playwright test.


So we can have one section on top that link to the chrome extension and then below that mention the usage of it in e2e tests.

Reply:
Also do mention that these functions are web-only

Reply:
I'll see hat else can be documented.